src/editorialreview/logic.py

```diff
@@ -9,8 +9,8 @@
 from django.db.models import Q
 from django.utils.encoding import smart_text
 
-from core import models as core_models
-from review import models as review_models, logic as review_logic
+from core import models as core_models, logic as core_logic
```

Comment:
best for readability reasons, we put this on separate lines for each new import like this. I had trouble finding the import at first

Reply:
That's how it's done throughout the codebase so I just stuck to that.